src/comments/comment.dto.ts

```diff
@@ -11,9 +11,6 @@ export class InsertCommentDto {
 
   @IsString()
   postId: string;
```

Comment:
This comment is for your comment.controller.ts since I can't view it in this PR since it isn't a recent change.

```
@Controller('comments')
export class CommentsController {
  constructor(private readonly commentsService: CommentsService) {}

  @Post()
  public async addComment(
    @Body() body: InsertCommentDto,
  ): Promise<{ id: string }> {
    const generatedId = await this.commentsService.insertComment(body); //change this name to just id
    return { id: generatedId }; // change this to return { id }
  }

  @Get()
  async getAllComments() { // where is the return type?
    const comments = await this.commentsService.getComments(); // simply this -> return await .... this.c
    return comments;
  }

  @Get(':id')
  getComment(@Param('id') commentId: string) { // where is the return type?
    return this.commentsService.getSingleComment(commentId);
  }

  @Patch(':id')
  public async updateComment(
    @Param('id') commentId: string,
    @Body() body: UpdateCommentDto,
  ): Promise<Comments> {
    return await this.commentsService.updateComment(commentId, body);
  }

  @Delete(':id')
  async removeComment(@Param('id') commentId: string) { // where is the return type?
    await this.commentsService.deleteComment(commentId);
    return null;
  }
}
```

Reply:
Check out the comments I added directly in the code block